backend/characterconnect/api/game_manager.py

```diff
@@ -151,7 +153,8 @@ def next_turn(self, board: List[List[str]]) -> int:
             if not valid_start(board):
                 raise InvalidStartException()
         # find the word on the Scrabble board, and check if it's valid.
-        word: str = find_word(self.old_board, board, self.turn)
+        game = GameManager(self.id)
```

Comment:
I misread your message earlier, you shouldn't create a new `GameManager` here. This will basically create a brand new game instance.

Since the find_word function is being called within an already existing GameManager instance, you can actually just pass `self` into the function which will pass the current instance in. I think this is normally quite bad style, since the function is quite impure it's got lots of side effects now, it would definitely be better to pass in a board and have the function return the word and then the tiles placed as-well since that wouldn't mutate the game. But we don't have a huge amount of time so I don't mind if you want to do it this way.